content/en/_index.md

```diff
@@ -5,79 +5,49 @@ description: "Documentation related to EGI activities"
 
 <!-- markdownlint-disable no-inline-html -->
 
-{{< blocks/cover title="Welcome to EGI Documentation!" image_anchor="top"
-    height="full" color="blue" >}}
-
-  <p class="lead mt-3">
-    This website hosts the first version of the new EGI documentation.<br />
-    It will be further improved with more sections in the coming months.<br />
-    <a href="https://github.com/EGI-Federation/documentation/issues/new">Your
-    feedback and suggestions are welcome!</a>
-  </p>
-
-<div class="mx-auto">
-  <a class="btn btn-lg btn-primary mr-3 mb-4" href="{{< relref "/users" >}}">
-    User Documentation <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-  </a>
-
-  <a class="btn btn-lg btn-primary mr-3 mb-4" href="https://www.egi.eu">
-    Learn More about EGI <i class="fas fa-arrow-alt-circle-right ml-2"></i>
-  </a>
-
-<a class="btn btn-lg btn-success mr-3 mb-4"
-href="{{< relref "/about/contributing" >}}"> Contribute
-<i class="fab fa-github ml-2 "></i> </a>
-
-  <p class="lead mt-2">EGI Documentation for users and service providers.</p>
-
-{{< blocks/link-down color="info" >}}
-
-</div>
-{{< /blocks/cover >}}
-
-<div class="mx-auto">
-{{< blocks/section color="primary" type="features">}}
-
-{{% blocks/feature icon="fa-user-friends" title="User Guides" url="/users" %}}
-Contains documentation of the EGI services and step-by-step guides on how to
-use them. {{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-users-cog" title="Service Providers"
-url="/providers" %}} Explains how to join the EGI infrastructure as a service
-provider to offer your innovative services. {{% /blocks/feature %}}
-
-{{% blocks/feature icon="fa-cogs" title="Internal Services" url="/internal" %}}
-Hosts the documentation of the EGI services that power the
-cloud services of EGI Federation. {{% /blocks/feature %}}
+{{< blocks/landing color="primary" height="min" title="Welcome to the EGI Documentation!">}}
+  <div>
+    <p class="lead mt-3">
+      <a href="https://www.egi.eu">EGI</a> is an international e-Infrastructure
+      that provides advanced computing and data analytics services for research
+      and innovation. The EGI Federation offers a wide range of services for compute,
+      storage, data and support.
+    </p>
+    <p class="lead mt-3">
+      This is the landing page for the user and provider documentation of the
```

Comment:
I'm not really fond of the "landing page" mention, couldn't we just replace by "This is the user and provider documentation of the EGI services"?

Reply:
Replaced it with "starting"